CONTRIBUTING.md

```diff
@@ -32,6 +32,42 @@ To get started we recommend:
 
 6. Install [golangci-lint] linter and run `make lint` to execute additional code linters.
 
+### Quick Start Local Development (MacOS)
+
+First install [colima](https://github.com/abiosoft/colima):
+
+```
+brew install colima
+```
+
+Start a colima VM with a single node Kubernetes cluster:
+
+```
+colima start --kubernetes
+```
+
+Then spawn a Docker container with a Go toolchain, with access to the Kubernetes cluster:
+
+```
+docker run -it --rm --network host -v $(pwd):/src -v $HOME/.kube:/root/.kube golang:1.19.5-bullseye bash
```

Comment:
Can this somehow be tied to the src tree so we don't have a chance of the go version drifting from what's actually in troubelshoot?

Reply:
good idea